include/mgp.py

```diff
@@ -145,6 +145,15 @@ def __init__(self, name: str):
 
     @property
     def name(self) -> str:
+        """
+        Get the name of the label.
```

Comment:
Gets? As for `Raises`, `Returns`
I am only writing the comment here, there might be other occurrences. Feel free to also close this comment if I am wrong, no need to ask for another review.